Agrispace/settings.py

```diff
@@ -145,4 +150,9 @@
 STATICFILES_DIRS = (
     os.path.join(BASE_DIR, 'staticfiles'),
 )
-CORS_ORIGIN_ALLOW_ALL = True
\ No newline at end of file
+CORS_ORIGIN_ALLOW_ALL = True
+cloudinary.config(
+    cloud_name = "dczmdkhbw",
```

Comment:
keys should put these keys in your environment variables:
`api_key`, `api_secret`, `cloud_name`